File: Server/monolithic-spring-boot/src/main/java/com/vserdiuk/tutorial/monolithicspringboot/model/Student.java
package com.vserdiuk.tutorial.monolithicspringboot.model;

import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

import javax.persistence.*;

@Data
@Entity
@Table(name = "students")
public class Student {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "student_id")
    @ApiModelProperty(notes = "The database generated student ID")
    private Long id;

    @Column(name = "first_name")
    @ApiModelProperty(notes = "Student first name")
    private String firstName;

    @Column(name = "last_name")
    @ApiModelProperty(notes = "Student last name")
    private String lastName;
}

File: Client/monolith-client/src/app/student.ts
export class Student {
  id: number;
  firstName: string;
  lastName: string;

  constructor(firstName: string, lastName: string, id?: number) {
    this.firstName = firstName;
    this.lastName = lastName;
    if (id) {
      this.id = id;
    }
  }
}
